### PR TITLE
feat(login): add municipality select to login

### PR DESCRIPTION
**Pull request by velrest**
The municipality is now saved with the login so we know to which municipality
a user belongs as this information is not provided by the gwr api.

BREAKING CHANGE: rename `municipalityId` to `municipality` in config

### CODE REVIEW

**Comment by velrest** [comment]
This pr depends on https://github.com/adfinis-sygroup/ebau-gwr/pull/19

File: addon/components/login-modal.js [hunk 1]
```diff
@@ -2,18 +2,26 @@ import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
+import config from "ember-ebau-gwr/config/environment";
```
Comment (kaldras) [inline addon/components/login-modal.js:5]: As discussed: For the sake of clarity we should use a different name so it is not confused with the `config` service.

**Review by kaldras — CHANGES_REQUESTED** [review]
Small change for clarity otherwise looking good! :+1:

File: addon/components/login-modal.js (hunk 1 above)
Comment (velrest) [reply addon/components/login-modal.js]: Done :+1:

**Review by kaldras — APPROVED** [review]

**Pull request merged**